modules/platform/nuxeo-automation/nuxeo-automation-features/src/main/java/org/nuxeo/ecm/automation/core/operations/services/FileManagerImport.java

```diff
@@ -82,7 +88,7 @@ public DocumentModel run(Blob blob) throws OperationException, IOException {
         DocumentModel currentDocument = getCurrentDocument();
         String path = currentDocument.getPathAsString();
         FileImporterContext fileCreationContext = FileImporterContext.builder(session, blob, path)
-                                                                     .overwrite(overwite)
+                                                                     .overwrite(overwite || overwrite)
```

Comment:
Couldn't this be dealt with the following code?
```
@Param(name = "overwrite", required = false, alias = "overwite")
protected Boolean overwrite = false;
```
Then we could have simply:
```
.overwrite(overwrite)
```

Reply:
It was the first idea, but what about the deprecation part then?

Reply:
Indeed, but it seems simpler to use aliases that were made for this purpose, and not have the deprecated member in the logic part.
I think it's OK that the compatibility is kept forever, or when we remove the deprecated member we could also simply remove the alias.

Reply:
I am +1 for that too :)